src/structures/Message.js

```diff
@@ -426,6 +426,26 @@ class Message extends Base {
     });
   }
 
+  /**
+   * Crossposts the message.
+   * @returns {Promise<Message>}
+   * @example
+   * // Crosspost a message
+   * if (message.channel.type === 'news') {
+   *    message.crosspost()
+   *      .then(msg => console.log('Crossposted message`))
+   *      .catch(console.error);
+   * }
+   */
+  async crosspost() {
+    await this.client.api
+      .channels(this.channel.id)
+      .messages(this.id)
+      .crosspost()
```

Comment:
```suggestion
      .crosspost
```
small nit but the brackets aren't needed

Reply:
They may not be needed in class constructors (`new Client` and `new Client()` are both valid), but it's not for this case. Besides, the lack of them would reduce readability and bring inconsistencies.

Reply:
removing them would also make it consistent with https://github.com/discordjs/discord.js/blob/5af1a552bc7ffde85e375d046888f4c0f4e69636/src/managers/MessageManager.js#L82, https://github.com/discordjs/discord.js/blob/5af1a552bc7ffde85e375d046888f4c0f4e69636/src/managers/MessageManager.js#L138, and so on

Reply:
Ah, my bad, I thought that was the example, but it was the API router 😅